pyaedt/edb_core/hfss.py

```diff
@@ -1024,6 +1024,84 @@ def create_lumped_port_on_net(
                 return edges_pts
         return True
 
+    @pyaedt_function_handler()
+    def create_vertical_circuit_port_on_clipped_traces(self, nets=None, reference_net=None, user_defined_extent=None):
+        """Create an edge port on clipped signal traces.
+
+        Parameters
+        ----------
+        nets : list, optional
+            String of one net or EDB net or a list of multiple nets or EDB nets.
```

Comment:
In line 1033, I think we need to indicate that the data type can be a string or list.